index.html

```diff
@@ -5,41 +5,48 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Personal Portfolio</title>
-    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="main.css">
     <style> @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap'); </style>
 </head>
 <body>
-    <header>
-        <section class="navSec">
+    <header class="header">
+        <div class="navSec">
             <div class="logo">
-                <h3>welcome</h3>
+                <a href="#">welcome</a>
             </div>
             <nav>
-                <img src="/images/Enabled.png" alt="navigation">
+                <img src="images/Enabled.png" alt="navigation">
             </nav>
-        </section>
-        <section class="banner">
-            <div class="headline">
-                <h1>Hey there. I’m Imran Hossain <br> <label>I’m a software developer</label> </h1>
-            </div>
-            <article>
-                <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                    don’t hestiate to contact me.</p>
-            </article>
-            <div class="contact">
-                <ul>
+        </div>
+    </header>
+    <section class="banner">
+        <div class="headline">
+            <h1>Hey there. I’m Imran Hossain <br> <label>I’m a software developer</label> </h1>
```

Comment:
- [ ] [OPTIONAL]  please take into consideration that The `<label>` element is used to associate a text label with a form `<input>` field. The label is used to tell users the value that should be entered in the associated input field. you should avoid using it here for a heading text, you can use a `<span>` tag if you want to select this text.